core/src/main/java/com/sequenceiq/cloudbreak/controller/mapper/ConstraintViolationExceptionMapper.java

```diff
@@ -44,4 +45,9 @@ Status getResponseStatus() {
         return ConstraintViolationException.class;
     }
 
+    private void addValidationError(ConstraintViolation<?> violation, ValidationResult validationResult) {
```

Comment:
This method is too short and has side effect, should be moved inside the `forEach` closure.